package.json

```diff
@@ -1,8 +1,35 @@
 {
   "name": "ethereumjs-vm",
   "version": "2.3.3",
-  "description": "an ethereum VM implementation",
-  "main": "index.js",
+  "description": "An Ethereum VM implementation",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "coverage": "istanbul cover ./tests/tester.js -- -s",
+    "coveralls": "npm run coverage && coveralls <coverage/lcov.info",
+    "testVM": "node ./tests/tester -v",
```

Comment:
This is lacking `build:dist`.

Reply:
VM tests are not active atm, this would only be pseudo-consistency.